skills/cairo-auditor/scripts/quality/detector_bridge.py

```
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+
+import importlib.util
+import os
+import re
+import sys
+from pathlib import Path
+from types import ModuleType
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)